lab_1_classify_by_unigrams/main.py

```diff
@@ -63,6 +108,32 @@ def detect_language(
     :param profile_2: a dictionary of a known profile
     :return: a language
     """
+    if not (
+            isinstance(unknown_profile, dict)
+            and isinstance(profile_1, dict) and isinstance(profile_2, dict)
+    ):
+        return None
+
+    name_1, name_2 = profile_1['name'], profile_2['name']
+    metric_1 = compare_profiles(unknown_profile, profile_1)
+    metric_2 = compare_profiles(unknown_profile, profile_2)
+
+    if (
```

Comment:
please reverse the condition and return None in `if`

Reply:
done